source/blog/2019-03-08-the-ember-times-issue-88.md

```diff
@@ -12,8 +12,10 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## Only a few days left to complete the [Ember Community Survey](https://emberjs.com/ember-community-survey-2019/)
 
+The *12th of March* is the deadline to complete the[Ember community survey](https://emberjs.com/ember-community-survey-2019/). Thanks a lot  if you have already answered the survey. Don’t forget to poke a lot of people around you to complete it too. 
```

Comment:
I think it would be 
```suggestion
The *12th of March* is the deadline to complete the [Ember community survey](https://emberjs.com/ember-community-survey-2019/). Thanks a lot  if you have already answered the survey. Don’t forget to poke a lot of people around you to complete it too. 
```